pkg/virtual/workspaces/registry/kubeconfig_rest_test.go

```diff
@@ -807,7 +807,7 @@ func TestKubeconfigFailBecauseShardNotFound(t *testing.T) {
 	test := TestDescription{
 		TestData: TestData{
 			user:  user,
-			scope: organizationScope,
+			scope: OrganizationScope,
```

Comment:
IMHO test fixtures should explicitly not use constants and helpers - if we are bound to an outward-facing API we need to know that a client's expected behavior will be broken if (for instance) I change this constant incorrectly.